src/services/auth.service.js

```diff
@@ -81,15 +94,66 @@ export const loginService = async body => {
   }
 };
 
+/**
+ * @method findUserById
+ * Queries the database to find a user using the provided id
+ * @param {number} userId
+ * @returns {object}  Database User Instance
+ */
+export const findUserById = async userId => await User.findByPk(userId);
+
 /**
  * @method isUserExist
  * - it persist a new user to the database
  * - returns a promise
- *
- * @param {String} userEmail user's email
- *
- * @returns {Promise}
  */
-
 export const isUserExist = async userEmail =>
   User.findOne({ where: { email: userEmail } });
+
+/**
+ * @method forgetPasswordServices
+ * - Helps generate password reset token and
+     sends mail to the user's email Address containing token and link
+ * @param {object} user - contains required user info to generate password reset token
+ * @returns {promise}
+ */
+
+export const forgotPasswordServices = async user => {
+  const token = await getPasswordResetToken(user);
+  const url = `${process.env.url}/api/v1/users/password_reset?token=${token}`;
```

Comment:
abstract `\api\v1` to a variable and read from there.

Reply:
alright